uqcsbot/channels.py

```diff
@@ -109,36 +126,67 @@ async def listchannels(self, ctx: commands.Context):
 
     @commands.command()
     @commands.has_permissions(manage_channels=True)
-    async def addjoinchannel(self, ctx: commands.Context, channel: discord.TextChannel):
+    async def addjoinchannel(self, ctx: commands.Context, channel: discord.TextChannel, emoji):
```

Comment:
Minor issue: `emoji` should ideally have a type, I think it being `str` should be fine